vscode-web/run.sh

```diff
@@ -37,20 +69,31 @@ for extension in "$${EXTENSIONLIST[@]}"; do
     continue
   fi
   printf "🧩 Installing extension $${CODE}$extension$${RESET}...\n"
-  output=$($VSCODE_SERVER --install-extension "$extension" --force)
+  output=$($VSCODE_WEB "$EXTENSION_ARG" --install-extension "$extension" --force)
   if [ $? -ne 0 ]; then
     echo "Failed to install extension: $extension: $output"
     exit 1
   fi
 done
 
-# Check if the settings file exists...
-if [ ! -f ~/.vscode-server/data/Machine/settings.json ]; then
-  echo "⚙️ Creating settings file..."
-  mkdir -p ~/.vscode-server/data/Machine
-  echo "${SETTINGS}" > ~/.vscode-server/data/Machine/settings.json
+if [ "${AUTO_INSTALL_EXTENSIONS}" = true ]; then
+  if ! command -v jq > /dev/null; then
+    echo "jq is required to install extensions from a workspace file."
+    exit 0
```

Comment:
Just occurred to me, should this be considered a failure?

```suggestion
    exit 1
```

Reply:
I am not sure. From a user perspective it would be nice to continue without an error and print a warning

Reply:
I guess there is where monitoring works when errors happen in a large deployment.

Reply:
Oh if `exit 1` prevents the workspace from starting then yeah, better to continue.

Reply:
it doesn't prevent the workspace from starting but does show a warning on the page that the script was not successful.

Reply:
Ahh gotcha yeah `exit 1` seems like the right move then.

Reply:
I think it's fine. If we don't fine jw we should just skip installing extensions and continue with the rest of the script.

Reply:
I have mixed feelings about it, because enabling autoinstall without `jq` is a bug the template author has to fix (I think, right?), and not showing the error could make it so no one notices.

On the other hand, it does seem annoying as a user that I cannot launch VS Code just because someone forgot to install `jq`. 

stderr gets highlighted in the terminal, right?  Maybe we output to stderr and then keep going.

Also if we use `node` instead, we avoid the problem entirely.

Reply:
I will try the node to in another PR.